.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -25,28 +23,22 @@ body:
       required: true
 
   - type: textarea
-    id: expectation
+    id: reproduction-steps
     attributes:
       label: How can we reproduce this?
       description: Thougougly explain how to reproduce the bug.
-    validations:
```

Comment:
Also removed several declarations of non-required validations, as this is the default behaviour